docusaurus/docs/learn/qos/2_evm_archival.md

```diff
@@ -116,6 +117,113 @@ The following methods require archive data when requesting information older tha
 
 Note: These methods can also be used for recent data (< 128 blocks old), but archive access is required when requesting older data.
 
+## Adding new EVM Archival Checks
+
+<!-- TODO_MOVE(@commoddity): Update this section and merge it into JUDGE docs once JUDGE PR is merged -->
+
+The process for adding new archival check configurations is somewhat manual but most be performed only once per chain.
+
+Configuration must be added to [`service_qos_config.go`](https://github.com/buildwithgrove/path/blob/main/config/service_qos_config.go).
+
+For more information on the `service_qos_config.go` file, see the [Service QoS Config](./1_supported_services.md) page.
+
+**Example**
+
+This example uses the `Polygon zkEVM` chain (`F029`). Most EVM chain block explorers use a similar format for their browser UI.
+
+- Block Explorer: https://zkevm.polygonscan.com/
+
+**Steps**
+
+1. Go to chain's block explorer and search for `Top Account` in the `Blockchain` dropdown
```

Comment:
Once this is in a new page, you can create subheaders for this.

It'll have a good ToC.

Reduced cognitive overhead.

User focuses on one or the other:
- Adding archival checks
- Learning how archival checks work.

Reply:
Ditto.